README.md

```diff
@@ -33,3 +33,16 @@ yarn start
 Open http://localhost:8000 to view it in the browser.
 
 The page will reload if you make edits.
+
+### Contributor Notes
+
+#### `@shoegazer_`
+* Had to do the following to get the above instructions to work:
+> First thing I would do is to lock the dependencies by removing all "^" from the package.json file. 
+After this, would install the dependencies using node version 18: `nvm use 18` and `yarn install`
+to build, I would use node version 16: `nvm use 16, yarn build and yarn start`
+
+* Tried the above on Windows and didn't work -- on Linux (Ubuntu) it did
+
+Filed https://github.com/threshold-network/website/issues/114
```

Comment:
What error have you ecnountered here?

I've tried to set it up based on the README and I've managd to do it without any problems on Node v16.16.0.

Reply:
I'll get a repro from you, but this is what @evandrosaturnino told me to do and it worked.  It did not just work out of the box.

Reply:
@michalsmiarowski I tried with a fresh local clone using node 16 and had no issues.  Reverted README and closed the github issue.